optaplanner-wb-webapp/src/main/resources/org/optaplanner/workbench/client/resources/i18n/AppConstants.properties

```diff
@@ -8,15 +8,12 @@ AdministrationPerspective.MenuNewRepository=New Repository
 AdministrationPerspective.MenuOrganizationalUnits=Organizational Units
 AdministrationPerspective.MenuManageOrganizationalUnits=Manage Organizational Units
 
-HomeProducer.KieKnowledgeDevelopmentCycle=The KIE Knowledge Development Cycle
-HomeProducer.Author=Author
-HomeProducer.FormalizeYourBusinessKnowledge=Formalize your Business Knowledge
-HomeProducer.ConfigureYourEnvironment=Learn how to configure your environment
-HomeProducer.Deploy=Deploy
-HomeProducer.ProjectAuthoring=Project Authoring
-HomeProducer.ArtifactRepository=Artifact Repository
-HomeProducer.Administration=Administration
-HomeProducer.DeployYourArtifacts=Deploy your Artifacts
+HomeProducer.Heading=Welcome to KIE Workbench
+HomeProducer.SubHeading=KIE Workbench offers a set of flexible tools, that support the way you need to work. Select a tool below to get started.
+HomeProducer.Design=Design
+HomeProducer.DesignDescription=Model, build, and publish your artifacts.
+HomeProducer.DevOps=DevOps
+HomeProducer.DevOpsDescription=Run and manage servers and active instances.
```

Comment:
Hi @jhrcek , could you comment that [here](https://issues.jboss.org/browse/BPMSPL-16) so the involved people can comment on that? In any case, I expect some changes on this screen in the near future, so this shouldn't be a blocker for this PR.